MapboxNavigation/NavigationViewController.swift

```diff
@@ -198,6 +198,17 @@ public protocol NavigationViewControllerDelegate: VisualInstructionDelegate {
      */
     @objc(navigationViewController:roadNameAtLocation:)
     optional func navigationViewController(_ navigationViewController: NavigationViewController, roadNameAt location: CLLocation) -> String?
+
+    /**
+     Called to allow the delegate to opt out of changes to UIApplication.shared.isIdleTimerDisabled
+
+     This method is called before NavigationViewController modifies UIApplication.shared.isIdleTimerDisabled. If not implemented, the proposed changes will be accepted automatically.
+
+     - parameter navigationViewController: The navigation view controller that will modify UIApplication.shared.isIdleTimerDisabled
+     - parameter to: The proposed new value of UIApplication.shared.isIdleTimerDisabled
+     - returns: true to accept the proposed change or false to reject it
+     */
+    @objc optional func navigationViewController(_ navigationViewController: NavigationViewController, shouldSetIsIdleTimerDisabledTo to: Bool) -> Bool
```

Comment:
Correct me if I'm wrong, but I think the double bool here is redundant. 

This could be rewritten as:

```swift
optional func navigationViewControllerShouldSetIsIdleTimerDisabledTo(_ navigationViewController: NavigationViewController) -> Bool
```

Reply:
The idea here was that `NavigationViewController` will pass in the value that it wants to change `isIdleTimerDisabled` to and then the delegate will return true or false to accept or reject that change.

Without the `to` parameter, the delegate wouldn't have enough information to know whether the `NavigationViewController` wants to enable or disable the idle timer.

Reply:
Okay, that makes sense.